official/vision/examples/starter/example_task.py

```diff
@@ -205,5 +205,5 @@ def validation_step(self,
     return logs
 
   def inference_step(self, inputs: tf.Tensor, model: tf.keras.Model) -> Any:
-    """Performs the forward step. It is used in validation_step."""
+    """Performs the forward step. It is used in validation step."""
```

Comment:
Don't change this as it's a function name, or use `validation_step` with quote

Reply:
Done.Thanks